deck/models.py

```diff
@@ -247,3 +247,13 @@ def create_initial_jury(sender, instance, signal, created, **kwargs):
     jury.users.add(instance.author)
     instance.jury = jury
     instance.save()
+
+
+@receiver(post_delete, sender=Proposal)
+def send_proposal_deleted_mail(sender, instance, **kwargs):
+    context = {'event_title': instance.event.title,
+               'proposal_title': instance.title}
+    message = render_to_string('mailing/jury_deleted_proposal.txt', context)
+    subject = _(u'Proposal from %s just got deleted' % instance.event.title)
+    recipients = [jury.email for jury in instance.event.jury.users.all()]
```

Comment:
The query that django does for create the user instance, which costs 1.12 seconds:
```
u'SELECT "auth_user"."id", "auth_user"."password", "auth_user"."last_login", "auth_user"."is_superuser", "auth_user"."username", "auth_user"."first_name", "auth_user"."last_name", "auth_user"."email", "auth_user"."is_staff", "auth_user"."is_active", "auth_user"."date_joined" FROM "auth_user" INNER JOIN "jury_jury_users" ON ( "auth_user"."id" = "jury_jury_users"."user_id" ) WHERE "jury_jury_users"."jury_id" = %s' - PARAMS = (u'31',)
```

The query that django does when you use the `value_list` approach, which costs 0.36 seconds:
```
u'SELECT "auth_user"."email" FROM "auth_user" INNER JOIN "jury_jury_users" ON ( "auth_user"."id" = "jury_jury_users"."user_id" ) WHERE "jury_jury_users"."jury_id" = %s' - PARAMS = (u'31',)
```

It is much more performatic than the other. o/